### ISSUE TITLE
Allow *.md generation for any chain (w/ retrieved info)

### ISSUE THREAD

**Issue by jacogr** [polkadot-js/api#5553]
Basic support for extended (not Substrate-only) metadata generation has been added in https://github.com/polkadot-js/api/pull/5552

The above adds some cli arguments to retrieve data and then use it and then allows generation for these.

What is missing is the following -

- RPCs are not generated based on the running chain (when overrides are applied)
- Runtime calls are not generated for the running chains (with overrides)

So basically, we need -

- where the argv endpoint is specified (as wss/ws), also retrieve version & RPCs, or
- where the endpoint is non-wss, also allow the specification of RPC/runtime JSON files, then
- supply these values to the RPC/runtime generation (as we do in the case for the Substrate/Polkadot/Kusama names)
